src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -28,7 +28,7 @@ export interface IDropdownSearchOwnProps extends React.ClassAttributes<DropdownS
   id: string;
   modMenu?: boolean;
   defaultOptions?: IDropdownOption[];
-  defaultSelectedOptions?: FixedQueue<IDropdownOption>;
+  defaultSelectedOption?: IDropdownOption;
```

Comment:
do you really need defaultSelectedOption since you can pass a selected option in your defaultOptions ?